src/test/java/cz/muni/fi/pa165/bluebat/entity/ExcursionTest.java

```diff
@@ -24,87 +25,81 @@
     @PersistenceUnit
     private EntityManagerFactory emf;
 
+    private void persistExcursion(Excursion excursion) {
+        EntityManager em = null;
+        try {
+            em = emf.createEntityManager();
+            em.getTransaction().begin();
+            em.persist(excursion);
+            em.getTransaction().commit();
+        } finally {
+            if (em != null) em.close();
+        }
+    }
+
+    private Excursion getFullyInitializedExcursion() {
+        Excursion result = new Excursion();
+        result.setName("testName");
+        result.setDuration(Duration.ofDays(1));
+        result.setDateFrom(LocalDate.now());
+        result.setDestination("testDestination");
+        result.setDescription("testDescription");
+
+        return result;
+    }
+
+    @DataProvider(name = "valuesNotBlank")
+    private static Object[][] getInvalidValues() {
+        return new String[][] { { null }, { "" }, { " " }, { "\t" }, { "\n" } };
+    }
+
+    @Test
+    public void excursion_fullyInitialized() {
+        Excursion excursion = getFullyInitializedExcursion();
+        Assertions.assertDoesNotThrow(() -> persistExcursion(excursion));
+    }
```

Comment:
Tento test nemá podle mého názoru význam, testuješ pouze metody frameworku

Reply:
Já se nad tím ještě zamyslel a asi to je v pohodě. Dalo by se argumentovat třeba tak, že testuješ, že tam nemáš nějaké anotace navíc, které by vyhazovaly exception na validních hodnotách.